app/javascript/components/CustomDomain.tsx

```diff
@@ -92,7 +92,7 @@ const CustomDomain = ({
           }}
         />
         {customDomain.trim() !== "" ? (
-          <Button className="pill" onClick={verifyCustomDomain} disabled={verificationInfo.buttonState === "verifying"}>
+          <Button className="inline-block align-middle !px-3 !py-2 bg-background text-foreground border border-border !rounded-[10rem] truncate shrink-0 -mr-2" onClick={verifyCustomDomain} disabled={verificationInfo.buttonState === "verifying"}>
```

Comment:
Does `<Pill asChild><Button>` work or does that cause the specificity issues you mentioned?